packages/core/src/PlayerContextProvider.js

```diff
@@ -70,15 +70,21 @@ const defaultState = {
 };
 
 // assumes playlist is valid
-function getGoToTrackState(prevState, index, shouldPlay = true) {
+function getGoToTrackState(
+  prevState,
+  index,
+  shouldPlay = true,
+  shouldForceLoad = false
```

Comment:
👍

Reply:
This is enough arguments that it might be better to send the arguments through in a hash, e.g.
```js
function getGoToTrackState({
  prevState,
  index,
  shouldPlay = true,
  shouldForceLoad = false
}) {
```

since it's getting hard to read.

Reply:
Should it al be a hash or just the optional args?, e.g.

```js
function getGoToTrackState(
  prevState,
  index,
  { shouldPlay = true, shouldForceLoad = false } = {}
) {
```

Reply:
I think everything should be at this point